cirrus/server/tests/test_feature_manifest.py

```diff
@@ -181,6 +181,111 @@ def test_compute_feature_configurations_targeting_doesnt_match(fml_setup):
     assert len(fml.merge_errors) == 0
 
 
+@pytest.mark.parametrize(
+    "targeting, targeting_context",
+    [
+        (
+            '(locale in ["en", "fa"])',
+            {
+                "clientId": "test",
+                "requestContext": {"locale": "en"},
+            },
+        ),
+        (
+            '(region in ["CA", "US"])',
+            {
+                "clientId": "test",
+                "requestContext": {"region": "US"},
+            },
+        ),
+        (
+            '(locale in ["en", "fa"]) && (region in ["CA", "US"])',
+            {
+                "clientId": "test",
+                "requestContext": {"locale": "en", "region": "US"},
+            },
+        ),
+    ],
+)
```

Comment:
@jeddai @jhugman I am testing targeting as it used SDK, I was hoping `locale` and `region` should work, but for some reason `region` is not working, although `locale` is working fine